Kernel/Unit.hpp

```diff
@@ -84,17 +84,26 @@ class Unit
   /** see isTheoryAxiom in Inference.cpp */
   bool isTheoryAxiom() const { return _inference.isTheoryAxiom(); }
 
-  /** return true if there is an input node in the deriviation  */
-  bool derivedFromInput() const;
-  /** return true if there is a node in the derivation that is derivedFromGoal
-      this is a current workaround for the fact that GS doesn't always set the
-      correct inputType **/
-  bool derivedFromGoalCheck() const;
-
```

Comment:
Does this functionality exist elsewhere now?

Reply:
Yes, done in one pass during 
```
bool seenInputInference = refutation->minimizeAncestorsAndUpdateSelectedStats();
```
in UIHelper.


`refutation->derivedFromGoal()` is correctly updated during the minimization.